### DISCUSSION TITLE
styles in body

### DISCUSSION THREAD

**Discussion by taymazmostafaei** [PHPMailer/PHPMailer#2303] [General]
hi how we can use css style in `$mail->Body` ?
can we use ?! 
`<style></style>`

**Comment by NiklasBr** [comment]
This article might be of help: [CSS in HTML emails: What you need to know to get started](https://content.myemma.com/blog/css-in-html-emails-what-you-need-to-know-to-get-started)

**Reply by taymazmostafaei** [reply]
thanks
